Client/Language.cs

```diff
@@ -0,0 +1,257 @@
+        public static string tbBoth = "Both (if installed)";
+
+        // In-Server Window
+        public static string chatBtn = "Chat";
+        public static string craftBtn = "Craft";
+        public static string debugBtn = "Debug";
+        public static string screenshotBtn = "Screenshot";
+        public static string timeNow = "NOW";
+        public static string inFuture = "{0} in the future";
+        public static string inPast = "{0} in the past";
+        public static string syncBtn = "Sync";
+        public static string pilotLabel = "Pilot: {0}";
+
+        // Craft Library
+        public static string craftLibrary = "Craft Library";
+        public static string plrCraftLibrary = "{0}'s Craft Library";
+        public static string uploadedCraftMsg = "Craft uploaded!";
+        public static string uploadingCraftMsg = "Uploading craft...";
+        public static string savedCraftMsg = "Craft {0} saved!";
+
+        // Screenshot Library
+        public static string screenshots = "Screenshots";
+        public static string sharedScreenshotMsg = "{0} shared a screenshot";
+        public static string screenshotUploadedMsg = "Screenshot uploaded!";
+        public static string uploadingScreenshotMsg = "Uploading screenshot...";
+        public static string downloadingScreenshotMsg = "Downloading screenshot...";
+
+        // Dates
+        public static string singularYear = "year";
+        public static string pluralYear = "years";
+        public static string singularMonth = "month";
+        public static string pluralMonth = "months";
+        public static string singularWeek = "week";
+        public static string pluralWeek = "weeks";
+        public static string singularDay = "day";
+        public static string pluralDay = "days";
+        public static string singularHour = "hour";
+        public static string pluralHour = "hours";
+        public static string singularMinute = "minute";
+        public static string pluralMinute = "minutes";
+        public static string singularSecond = "second";
+        public static string pluralSecond = "seconds";
+        public static string shortYear = "y";
+        public static string shortMonth = "m";
+        public static string shortWeek = "w";
+        public static string shortDay = "d";
+
+    }
+
+    public enum DMPLanguage
+    {
+        ENGLISH,
+        SPANISH,
+        BRAZILIAN,
+        GERMAN
+    }
+
+    public class LanguageWorker
+    {
+        //LanguageWorker
+        private static LanguageWorker singleton = new LanguageWorker();
+        private string dataLocation;
+        private string langFile;
+        private string LANGUAGE_FILE = "english.txt";
+
+        private bool loadedSettings = false;
+
+        private static Dictionary<string, string> languageStrings;
+
+        public static LanguageWorker fetch
+        {
+            get
+            {
+                return singleton;
```

Comment:
DMP uses the singleton approach in many places, so does KSP, and so does unity ;).

The only real contender we have for singletons is a [facade](http://en.wikipedia.org/wiki/Facade_pattern), but meh, that's just a singleton of singletons.